src/nn/wasm/src/bind/src/binding.cpp

```diff
@@ -18,12 +21,22 @@ using namespace tflite;
 
 namespace binding_utils {
   static gemmlowp::GemmContext gemm_context;
+  static CpuBackendContext cpu_backend_context;
+  static CpuFlags cpu_flags;
   
   // help functions
   void set_gemm_context_threads_num(int threads_num) {
     gemm_context.set_max_num_threads(threads_num);
   }
 
+  void set_cpu_context_threads_num(int max_num_threads) {
+    cpu_backend_context.SetMaxNumThreads(max_num_threads);
+  }
+
+  void get_cpu_flags(CpuBackendContext* cpu_backend_context, CpuFlags* cpu_flags) {
```

Comment:
Do we need to expose this function to JS? I can only find it used by ops. If so, why not call `tflite::GetCpuFlags` directly?